templates/terraform/command-center-project/cloudsql.tf

```diff
@@ -13,8 +13,8 @@ module cloudsql {
     role = "database"
     state = "active"
   }
-  db_names = []
-  user_names = []
+  db_names = ["argo", "airflow", "datarepo", "stairway"]
+  user_names = ["argo", "airflow", "datarepo"]
```

Comment:
(Re-thinking the ticket description, sorry) I don't think we'd want to hard-code "datarepo" or "stairway" within the command-center module, because those only apply in dev. I can think of a few ways around it:
1. Have these be args to the command-center module. There will be some repetition between dev and prod, but it'll be easy to understand at the call site.
2. Have args for "extra" DBs and users. Less repetition, but maybe weird at the callsite.
3. Hard-code only the common pieces here, and have the dev module create more DBs / users out-of-band. Cleanest for this module, but messy from the outside.

I'm not sure which option is least-bad, am curious to hear what others think.

Reply:
I think option (1) is clean and would make sense to us 6 months from now, and the small amount of repeated code would be worth the clarity. I think "extra" DBs and users would be confusing, so I'm not big on (2). I think (3) makes sense too, so I guess I'm opinionated on not going the way of (2) and am fine with either other one.

Reply:
I agree that option 1 and 3 would be the cleanest, although I don't know how messy (or not messy) it would be to create DBs/users out-of-band without digging into it a bit.

Do you have a preference @danxmoran ?

Reply:
I like options 1 and 2 but leaning towards 1.
1. Having these all be arguments would mean the dev and prod calls would have a similar structure which should reflect in an easier call and easier differentiation between the two.
2. Having args for extra parameters would also solve this problem but would require some additional logic for the extra work that would be required for the dev module.

Reply:
Sounds good to me! I'll go with option 1 then unless anyone objects 😁

Reply:
Just made the changes. Let me know if it's what you all had in mind